auditor/src/client/mod.rs

```diff
@@ -146,8 +209,7 @@ impl AuditorClient {
             .header("Content-Type", "application/json")
             .json(record)
             .send()
-            .await
-            .map_err(ClientError::ReqwestError)?;
+            .await?;
 
         if response.text().await.map_err(ClientError::ReqwestError)? == ERR_RECORD_EXISTS {
```

Comment:
This `map_err` can be removed as well, right?

Reply:
It already has been ;-)